nilearn/reporting/tests/test_sphinx_report.py

```diff
@@ -19,8 +20,9 @@ def _gen_report():
     return report
 
 
-def test_scraper(tmpdir):
+def test_scraper():
     """Test report scraping."""
+    tmpdir = tempfile.mkdtemp()
```

Comment:
`mkdtemp` does not remove itself.

```python
with tempfile.TemporaryDIrectory() as tmpdir:
```
You might have to manually delete still in memory objects manually before the context exits for tests in Windows, if they raise an error.